feed/index.html

```diff
@@ -7,6 +7,8 @@
     <title>Feed Page</title>
     <link rel="stylesheet" href="../dist/css/index.css">
     <script src="../dist/js/bundle.js"></script>
+    <script src="../src/js/loggedIn.js"></script>
+    <script src="../src/js/feed.js" type="module"></script>
```

Comment:
Should be using modules (imports/exports) rather than linking to multiple files.